errbot/botplugin.py

```diff
@@ -266,6 +266,19 @@ def callback_message(self, message: Message) -> None:
         """
         pass
 
+    def callback_mention(self, message: Message, mentioned_people: Sequence[Identifier]) -> None:
+        """
+            Triggered if there are mentioned people in message.
```

Comment:
If we're going to merge this, I think we should make an effort to have it available on all backends.

This docstring should also get updated to reflect that this may not be implemented on all backends, so that people aren't left wondering why it isn't working on other backends.